mpsl/Kconfig

```
@@ -33,6 +33,10 @@ config MPSL_LIB_DIR
 	default "nrf52" if SOC_SERIES_NRF52X
 	default "nrf53" if SOC_NRF5340_CPUNET
 	default "nrf54h" if SOC_NRF54H20_CPURAD
+	default "nrf54LM_SERIESl" if SOC_NRF54LM20A
```

The other variants do not have "series" in their folder name
```suggestion
	default "nrf54lm" if SOC_NRF54LM20A
```

---

yes, thanks. was a typo